.github/workflows/push.yml

```diff
@@ -480,3 +483,14 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build and Push fio image
         run: make fio
+  
+  litmus-pg-jmeter:
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.litmus-pg-jmeter == 'true'
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Building litmus-pg-jmeter image
+        run: make litmus-pg-jmeter
```

Comment:
Can we directly use `make _push_litmus_pg_jmeter` here, as it'll do both build and push multiarch.

Reply:
Thank you @uditgaurav, Updated PTAL